core/src/main/java/io/kestra/core/repositories/FlowRepositoryInterface.java

```diff
@@ -162,4 +162,6 @@ default List<String> findDistinctNamespace(String tenantId, String prefix) {
     FlowWithSource update(GenericFlow flow, FlowInterface previous) throws ConstraintViolationException;
 
     FlowWithSource delete(FlowInterface flow);
+
+    Boolean existForTenant(String tenantId);
```

Comment:
```suggestion
    Boolean existAnyNoAcl(String tenantId);
```

We always include `NoAcl` in the name of methods that bypasses ACLs

Reply:
make sense, done